cardano/base/views.py

```diff
@@ -7,17 +7,27 @@ def index(request):
     countries = Country.objects.order_by('name', '-created').distinct('name')
     mozillians = Mozillian.objects.latest()
 
-    previous_week = Mozillian.objects.order_by('-created')[1]
-    vouched_diff = mozillians.vouched - previous_week.vouched
-    total_diff = mozillians.total - previous_week.total
+    try:
+        previous_week = Mozillian.objects.order_by('-created')[1]
+        vouched_diff = mozillians.vouched - previous_week.vouched
```

Comment:
Please remove any additional code from the `try` block. Only the code that can produce an exception should be here.